lib/loadable_config.rb

```diff
@@ -39,25 +39,42 @@ def self.attributes(*attrs, type: :string, optional: false)
   end
 
   def initialize
-    unless File.exist?(self.class._config_file)
-      raise RuntimeError.new("Cannot configure #{self.class.name}: configuration file '#{self.class._config_file}' missing")
+    prefix = LoadableConfig::Options.config_path_prefix || ''
+    subkey = LoadableConfig::Options.subkey || nil
+
+    prefix.chomp('/').concat('/') unless prefix.empty?
+    subkey = subkey.to_s          unless subkey.nil?
```

Comment:
One other option would be rather than simple setters/getters, something like a `LoadableConfig.configure do |config|  ...` block, similar to a few other gems we use